handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslServerContext.java

```diff
@@ -153,6 +153,17 @@ static ServerContext newSessionContext(ReferenceCountedOpenSslContext thiz, long
                 } else {
                     SSLContext.setCertVerifyCallback(ctx, new TrustManagerVerifyCallback(engineMap, manager));
                 }
+
+                X509Certificate[] issuers = manager.getAcceptedIssuers();
+                if (issuers != null && issuers.length > 0) {
+                    long bio = 0;
+                    try {
+                        bio = toBIO(issuers);
+                        SSLContext.setCACertificateBio(ctx, bio);
```

Comment:
did we want to use the return value ... or have this method throw if something bad happens?